source/reference/minio-mc/mc-support-perf.rst

```diff
@@ -165,6 +179,19 @@ Syntax
                       [--airgap]             \
                       ALIAS
 
+.. mc-cmd:: site-replication
+   :fullpath:
+
+   Measure the network throughput of site replication operations between configured peers.
```

Comment:
OK *this* implies that line 115 should be "...speed of site replication operations for all peer sites of ``minio1`` or something

Reply:
Paralleled the earlier line.